packages/@okta/vuepress-theme-prose/const/navbar.const.js

```diff
@@ -680,6 +690,16 @@ export const guides = [
           }
         ],
       },
+      {
+        title: "Manage orgs",
```

Comment:
Manage orgs be hidden, too - because when I look at it in the preview - it shows just Manage orgs with an expand icon, but it doesn't expand and when you click Manage orgs - you can access the guide....is that on purpose?